DEPENDENCIES.md

```diff
@@ -12,7 +12,7 @@
 | Tekton Triggers               | 0.20.2                                                                                                                                | Managed by Operator                                                          | Controller by OpenShift Pipelines Operator |
 | Pipelines as Code             | 0.10.2                                                                                                                                | Managed by Operator (future). Currently, manually installed                  | Upgrades to next versions need to be tested and approved |
 | Tekton Results                | 0.4.0                                                                                                                                 | Modified version of Results installed and maintained by Pipeline Service     | Upgrades to next versions need to be tested and approved |
-| Tekton Chains                 | ghcr.io/hacbs-contract/chains/controller</br>:poc-tep-84@sha256:853fb18d56f27bbc2747954</br>291b8a22a8bb58a37fd501eb9b0a29630723ed6b1 | Modified version of Chains installed and maintained by Pipeline Service team | Custom version required until https://github.com/tektoncd/chains/pull/436 is merged upstream |
+| Tekton Chains                 | 0.13.0                                                                                                                                | Modified version of Chains installed and maintained by Pipeline Service team | Upgrades to next versions need to be tested and approved |
```

Comment:
I added this to match what it says about Tekton Results. We have changes that are only upstream. Eventually, these will make their way into OpenShift Pipelines. At that point, we should stop using the upstream version in favor of the one provided by Red Hat.

Let me know if you'd like some of this written in the file.